db/migrate/20190712204915_add_gender_field_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddGenderFieldToUser < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :gender, :integer
```

Comment:
Are you allowing null values for gender? Maybe you can add a null constraint to avoid third state objects